Rakefile

```diff
@@ -28,12 +28,14 @@ RuboCop::RakeTask.new
 
 desc "Pull the latest versions of all dependencies into the gem for distribution"
 task :vendor_deps do
-  TEST_TRACK_JS_CLIENT_VERSION = '1.3.8'.freeze
+  TEST_TRACK_JS_CLIENT_VERSION = '2.0.0-alpha.2'.freeze
   TEST_TRACK_CLI_VERSION = 'v1.0.2'.freeze
 
   # Bundle JS client
+  sh 'npm init -y'
```

Comment:
Newer versions of `npm` require a `package.json` to be present when installed packages, so this temporarily creates one to be removed after the new client is copied into the vendored assets.